tests/integration-tests/test_runner.py

```diff
@@ -432,13 +440,16 @@ def _is_url(value):
         raise argparse.ArgumentTypeError("'{0}' is not a valid url".format(value))
 
 
-def _test_config_file(value):
-    _is_file(value)
+def _test_config_file(test_config_file, test_config_args=None):
+    _is_file(test_config_file)
     try:
-        config = read_config_file(value)
+        if test_config_args:
+            config = read_config_file(test_config_file, **test_config_args)
+        else:
```

Comment:
this if/else shouldn't be required since read_config_file is able to manage the args even if it's None.

Reply:
I would get an error like 
`** must be a mapping, not NoneType` for not putting the condition